apps/api/src/domain/types/AggregateCompositeIdentifier.ts

```diff
@@ -0,0 +1,25 @@
+import { isNullOrUndefined } from '../utilities/validation/is-null-or-undefined';
```

Comment:
Nice!

Reply:
Actually not. I need to break this out into our validation lib instead!